packages/static-hosting/lib/static-hosting.ts

```diff
@@ -547,6 +582,46 @@ export class StaticHosting extends Construct {
       }
     }
 
+    const cspPaths = props.cspPaths || [];
+    const cspRemapPaths = cspPaths.map(cspPath => {
+      const { path, indexPath, reportUri, fallbackCsp } = cspPath;
+
+      const requestFunction = new RequestFunction(
+        this,
+        "AlternativePathFunction",
```

Comment:
The names of constructs here will need the index of the array, otherwise if you have more than 1 element in `cspPaths` it will error as the construct already exists.